### DISCUSSION TITLE
Uncaught ReferenceError: Alpine is not defined

### DISCUSSION THREAD

**Discussion by zakmcrbit** [livewire/livewire#8767] [7. Bugs (archived)]
### Livewire version

3.2.6

### Laravel version

10.0.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. set config/livewire.php inject_assets to false and show_progress_bar to false.
2. manually implement @livewireStyles and @livewireScripts

### Repository or Wirebox URL

https://wirebox.app/b/4jn01

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by juliangarcess** [comment]
Hi @blursolo! your wireguard demo is private.

**Comment by zakmcrbit** [comment]
I'm sorry about that but the demo doesn't hold any useful information because I couldn't access the config file in wirebox, yet it is a mandatory field so I just created an empty example.

**Reply by juliangarcess** [reply]
ok! so please create a fresh laravel project that has that error and share the repo

**Comment by BorisKM** [comment]
It seems that problem are with this config key: `show_progress_bar`
Just keep it as `true` and hide `nprogress` in CSS
